Loop/Managers/WatchDataManager.swift

```diff
@@ -437,8 +443,14 @@ extension WatchDataManager: WCSessionDelegate {
             }
         case GlucoseBackfillRequestUserInfo.name?:
             if let userInfo = GlucoseBackfillRequestUserInfo(rawValue: message) {
-                deviceManager.glucoseStore.getCachedGlucoseSamples(start: userInfo.startDate.addingTimeInterval(1), end: nil) { (values) in
-                    replyHandler(WatchHistoricalGlucose(with: values).rawValue)
+                deviceManager.glucoseStore.getSyncGlucoseObjects(start: userInfo.startDate.addingTimeInterval(1)) { (result) in
```

Comment:
At some point we should talk about the watch using query anchors, now that we have that facility.  That's probably a bigger thing than we want to address here, but I think using anchors is probably the right way to do sync for the watch.

Reply:
Yes, make sense, for later.